components/alertbanner/package.json

```
@@ -25,27 +25,35 @@
 	},
 	"main": "dist/index.css",
 	"peerDependencies": {
-		"@spectrum-css/button": ">=13",
-		"@spectrum-css/closebutton": ">=5",
-		"@spectrum-css/divider": ">=3",
-		"@spectrum-css/icon": ">=7",
-		"@spectrum-css/tokens": ">=14 || >=15"
+		"@spectrum-css/button": ">=14.0.0 <15.0.0",
+		"@spectrum-css/closebutton": ">=6.0.0 <7.0.0",
+		"@spectrum-css/divider": ">=5.0.0 <6.0.0",
+		"@spectrum-css/icon": ">=9.0.0 <10.0.0",
+		"@spectrum-css/tokens": ">=16.0.0 <17.0.0"
 	},
 	"peerDependenciesMeta": {
+		"@spectrum-css/button": {
+			"optional": true
+		},
+		"@spectrum-css/closebutton": {
+			"optional": true
+		},
+		"@spectrum-css/divider": {
+			"optional": true
+		},
 		"@spectrum-css/icon": {
 			"optional": true
+		},
+		"@spectrum-css/tokens": {
+			"optional": true
```

```suggestion
		"@spectrum-css/tokens": {
			"optional": true
```
Tokens shouldn't be appearing as optional should it? (appearing on multiple components)

---

So it's not optional to appear in the DOM with the component but it is optional whether downstream consumers choose to install it as a dependency in the same package. It's problematic for us to list it as a required peerDep because SWC ships those styles from different packages. I definitely don't want to loose the peerDependency information but thought this would be a good way to reduce errors in the install that just end up ignored.